docs/src/documentation/05-development/03-hooks/usrandomid.mdx

```diff
@@ -0,0 +1,10 @@
+---
+title: useRandomId
+description: Generates unique random id
+redirect_from:
+  - /utilities/userandomid/
+---
+
+import UseRandomId from "@kiwicom/orbit-components/src/hooks/useRandomId/README.md";
```

Comment:
@mainframev the documentation should mention real use case for the hook, currently it looks like a component, but it's not.

Reply:
Oh, true. Sorry, I didn't realize it's imported README 🤦 

However, there's missing whitespace for the heading and thus resulting in wrong render:
https://github.com/kiwicom/orbit/pull/3055/files#diff-3ed8cb5d7c9c8fdbc5da47586e5a7f83decff66e90e3017399d4a2782a27fd5cR1

https://beta.orbit.kiwi/development/hooks/usrandomid/